code/game/machinery/machinery.dm

```diff
@@ -466,10 +466,8 @@ Class Procs:
 /obj/machinery/tesla_act(var/power)
 	..()
 	if(prob(85))
-		emp_act(2)
+		explosion(src.loc,1,2,4,flame_range = 2, adminlog = 0, smoke = 0)
```

Comment:
didn't people not want this to be a copy paste of the singlo? I get boosting its devastation but making it explode stuff pretty much every time it zaps would be kind of lame